documentation/tutorials/get-started.md

```diff
@@ -354,17 +357,19 @@ ticket
 
 ### Querying without persistence
 
-So far, there is no persistence happening. All that this simple resource does is return the record back to us. You can see this lack of persistence by attempting to use a `read` action:
+So far, we haven't stored any data in a database. All that this simple resource does is return the record back to us. You can see this lack of persistence by attempting to use a `read` action:
 
 ```elixir
 Helpdesk.Support.read!(Helpdesk.Support.Ticket)
 ```
 
 Which will raise an error explaining that there is no data to be read for that resource.
 
-In order to add persistence, we need to add a data layer to our resources. Before we do that, however, lets go over how Ash allows us to work against many different data layers (or even no data layer at all). Resources without a data layer will implicitly be using `Ash.DataLayer.Simple`, which will just return structs and do no persistence. The way that we do this is by leveraging `context`, a free-form map available on queries and changesets. The simple data layer looks for `query.context[:data_layer][:data][resource]`. It provides a utility, `Ash.DataLayer.Simple.set_data/2` to set it.
+In order to save our data somewhere, we need to add a data layer to our resources. Before we do that, however, lets go over how Ash allows us to work against many different data layers (or even no data layer at all).
```

Comment:
Data layers aren't necessarily *only* for saving data. They could be used as the back-end interactor, i.e if you wanted to send requests to an external system in response to create/update/destroy actions. I know that can be a bit of a heady concept for new people, but we want to make sure that people don't get the impression that Ash is for "CRUD persistence"

Reply:
Agree. Persistence and saving data are basically interchangeable though